docs/getting-started.md

```diff
@@ -3,126 +3,220 @@ id: getting-started
+- **Base Sepolia**
+- **Arbitrum Sepolia**
+- **Optimism Sepolia**
 
-- **CounterAppGateway:** `CounterAppGateway` is an `AppGateway`. It is a contract deployed on EVMx and not onchain. It dictates how the onchain contracts are called, composed, deployed and initalized. You can read more about chain abstracted deployments [here](/deploy). In this example when someone calls the `incrementCounters` function, it internally triggers calls to `increase` function on each provided instance. This is an [onchain write](/call-contracts) triggered from AppGateway. You can also [make read calls](/read) to the chains to use their state (see `readCounters`).
+This single command deploys and initializes `Counter` contracts across all target chains:
 
-[↘ Learn more about how to build applications on SOCKET](/writing-apps#architecture-overview)
+```bash
+forge script script/counter/DeployOnchainCounters.s.sol --broadcast --skip-simulation --legacy --with-gas-price 0
+```
+
+What happens behind the scenes:
+
+The deployment process follows these steps:
+
+1. **Initialization**: The script calls `deployContracts()` on the CounterAppGateway.
+
+2. **Promise Creation**: An `AsyncPromise` contract is created to track the completion status of the deployment.
+
+3. **Queueing**: The deployment request is queued in a `DeliveryHelper` contract on EVMx.
+
+4. **Payload Processing**: Queued items are processed into `PayloadDetails` and submitted for auction.
+
+5. **Transmitter Bidding**: Offchain [Transmitters](/transmitters) bid to execute the requested deployments.
+
+6. **Watcher Attestation**: Once a winning bid is selected, the watcher attests it on the Fast [Switchboard](/switchboards) on the target chain.
+
+7. **Onchain Execution**: The [Transmitter](/transmitters) executes the deployment on the target chain through the SOCKET contract, which verifies with the [Switchboard](/switchboards) that the Transmitter is authorized to execute the payload.
+
+8. **Event Observation**: The [Watcher](/watchers) observes the execution events and resolves the `AsyncPromise` with the deployment result.
+
+9. **Callback Execution**: This triggers the callback in the `AppGateway`, which:
+   - Stores the deployed contract address
+   - Deploys a Forwarder contract (an EVMx representation of the onchain deployed contract)
+   - Sets up the Forwarder to handle future multi-chain calls
+
+:::note Note
+The Fast Switchboard used here is just one example of a simplified [Switchboard](/switchboards) implementation that allows a trusted [Watcher](/watchers) to attest that a given [Transmitter](/transmitters) is authorized to execute a specific payload. Switchboards can be customized to handle various verification methods and proof systems depending on your security requirements.
+:::
+
+<div style={{ display: 'flex', justifyContent: 'center' }}>
+    <img src="/img/deploy_flow.png" alt="switchboard diagram" style={{ width: '100%' }} />
```

Comment:
Could you make this an svg without background? It is more maintainable and easier to edit if needed

Reply:
yup wasnt sure if we leave it in. thx